src/cdi_ndi/test/CMakeLists.txt

```diff
@@ -20,9 +20,13 @@ set( test_sources
 # Build Unit tests
 # ---------------------------------------------------------------------------- #
 
+# Both tests read a local test data file, 'ndi_data'. To avoid resource issues,
+# tell ctest to avoid running these two test at the same time with the keyword
+# RESOURCE_LOCK.
 add_scalar_tests(
   SOURCES "${test_sources}"
-  DEPS    "Lib_cdi_ndi" )
+  DEPS    "Lib_cdi_ndi"
+  RESOURCE_LOCK "ndi_data" )
```

Comment:
I was wondering why one of these tests occasionally failed, thanks for finding this!